frontend/packages/core-api-client/src/client/Form.ts

```diff
@@ -32,4 +34,22 @@ export class FormClient {
       200,
     );
   };
+
+  getAncestors = async (formId: string): Promise<FormDefinition[]> => {
+    const formResponse = await this.get({ id: formId });
+
+    if (!formResponse.response) {
+      throw new Error(formResponse.error);
+    }
+
+    const key = formResponse.response.fields.find(
+      (field: FieldDefinition) => field.key,
+    );
+
+    if (key && key.fieldType.reference) {
```

Comment:
```
    const referenceKey = formResponse.response.fields.find(
      (field: FieldDefinition) => field.key && key.fieldType.reference,
    );

    if (referenceKey) {
```

Reply:
kinda thought, since recipient are required to have exactly 1 key, which has to be a reference, this might be enough, but better save than sorry :heavy_check_mark: